src/components/ha-yaml-editor.ts

```diff
@@ -44,14 +44,14 @@ export class HaYamlEditor extends LitElement {
 
   @internalProperty() private _yaml = "";
 
-  @query("ha-code-editor", true) private _editor?: HaCodeEditor;
+  @query("ha-code-editor") private _editor?: HaCodeEditor;
 
   public setValue(value): void {
     try {
       this._yaml = value && !isEmpty(value) ? safeDump(value) : "";
     } catch (err) {
       // eslint-disable-next-line no-console
-      console.error(err);
+      console.error(err, value);
       alert(`There was an error converting to YAML: ${err}`);
```

Comment:
Should be converted to `showAlertDialog` with localize and include the value in it, and remove the console statement

Reply:
I'll do a follow up PR.